examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
This flag should be set in toolchain since it affects compatibility. I'm planning to add examples with in depth explanation in future: https://github.com/ruslo/hunter/issues/32. Toolchain `default` should fail and excluded from testing and uploading. This will show the fact that some of the library in system require C++11 support. At least please add:
```cmake
string(COMPARE EQUAL "${CMAKE_TOOLCHAIN_FILE}" "" no_toolchain)
if(no_toolchain)
  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
endif()
```
Also note that you need to check that flag is appropriate for the current compiler (e.g. `-std=c++11` is not for MSVC). Probably it's better to use abstraction [CMAKE_CXX_STANDARD](https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html):
```cmake
string(COMPARE EQUAL "${CMAKE_TOOLCHAIN_FILE}" "" no_toolchain)
if(no_toolchain)
  set(CMAKE_CXX_STANDARD 11)
endif()
```